event-adapter/adapter.go

```diff
@@ -23,15 +23,21 @@ import (
 	"k8s.io/apiserver/pkg/util/logs"
 )
 
+var (
+	maxRetrievedEvents = flag.Int("max-retrieved-events", 100, "Maximum number of events can be retrieved")
+	sinceMillis        = flag.Int64("retrieve-events-since-millis", -1, "Retrieve only events that where logged at most the given amount of seconds ago")
```

Comment:
Please update a message - meaning of this flag is a little different as we discussed.

Reply:
Updated